URBNSwiftCarousel/Classes/URBNCarouselZoomableCellCollectionViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  URBNCarouselZoomableCellCollectionViewCell.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 4/26/16.
+//
+//
+
+import UIKit
+
+/**
+ 
+ This subclass of UICollectionViewCell has a zoomable UIImageView on top of a scrollView.
+ 
+*/
+public class URBNCarouselZoomableCell: UICollectionViewCell, UIScrollViewDelegate {
```

Comment:
In the existing/legacy carousel, I always found it was really pain full to subclass these cells especially when we wanted to add additional content beyond the `imageView` and `scrollView`.

This might be a stretch, but can we do this in a more protocol-driven manner so there's greater flexibility in how the client wants to layout it's contentview?